bin/lib/create.js

```diff
@@ -272,7 +272,7 @@ exports.create = function (project_path, config, options, events) {
     // Make the package conform to Java package types
     return exports.validatePackageName(package_name)
         .then(function () {
-            exports.validateProjectName(project_name);
+            return exports.validateProjectName(project_name);
```

Comment:
```suggestion
            return validateProjectName(project_name);
```

(maybe)

Reply:
We can't do that, since calling the function on the `exports` object is necessary for stubbing via `spyOn` to be possible. The tests actually make use of that too.

`spyOn(create, 'validateProjectName')` in the spec file would stub `exports.validateProjectName` whereas `validateProjectName` would still refer to the original unstubbed function.